controllers/admin/AdminImagesController.php

```diff
@@ -175,6 +175,15 @@ public function __construct()
                         'desc' => $this->trans('Enable to optimize the display of your images on high pixel density screens.', [], 'Admin.Design.Help'),
                         'visibility' => Shop::CONTEXT_ALL,
                     ],
+                    'PS_IMAGEOPT_LAZY_RESIZE' => [
+                        'type' => 'bool',
+                        'title' => $this->trans('Resize missing thumbnails on demand', [], 'Admin.Design.Feature'),
+                        'required' => false,
+                        'is_bool' => true,
+                        'hint' => $this->trans('Resize the thumbnails when they are requested by HTTP, not during generation of HTML page.', [], 'Admin.Design.Help'),
+                        'desc' => $this->trans('If you have missing thumbnails, this can speed up page generation time by orders of magnitudes.', [], 'Admin.Design.Help'),
```

Comment:
"this can speed up page generation time by orders of magnitudes"

it sounds a bit like advertisement ^^ I think we might stay a bit more 'down to earth' in what we say

Anyway I don't decide, it's @Julievrz who decides wordings

Reply:
Sorry for the late answer, I think I missed this PR.

You're right. Maybe we should only keep "If you have missing thumbnails, this can speed up the page load time." wdyt?

Reply:
Sure. I make the correction.

I think my setup is kinda special. I chose this wording as a reaction of being tired of waiting literally for minutes for pages to display, and discovering that the culprit was this issue.